website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -1,18 +1,14 @@
 ## Spacing
 
-Spacing between sections, fields, text, and other form elements should follow a decreasing scale on an 8px grid. This creates a consistent vertical rhythm between elements and reinforces a natural hierarchy and association between elements.
+Spacing between sections, fields, text, and other form elements should follow a decreasing scale on an 8px grid. This creates a consistent visual hierarchy between elements.
 
 ### Form
 
-A **form** acts as a layout mechanism by wrapping the content, fields, and actions to apply consistent spacing while simultaneously handling the logic for submitting information and data collected by the fields.
-
-As a layout mechanism, a form consisting of more than one section should use a 32px gap in between sections.
-
-![Form container](/assets/patterns/form-patterns/form-container.png =450x*)
+A **form component** acts as a layout mechanism by wrapping the content, fields, and actions into sections with consistent spacing while simultaneously handling the logic for submitting information and data collected by the fields.
```

Comment:
```suggestion
A form acts as a layout mechanism by wrapping the content, fields, and actions into sections with consistent spacing while simultaneously handling the logic for submitting information and data collected by the fields.
```

Reply:
It seems kind of strange to me to talk about the Form component behavior like this in the context of the form patterns. Maybe say something simpler in this context like:

```
A form acts as a container for form sections, fields, and other form content.
```

Or perhaps delete the whole "Form" section here entirely.

What do you think? @heatherlarsen

Reply:
+1. This was in the original content, but "component" got added, so I suggested removing that, as I agree that it's strange since it's not a component, but an HTML element. I'd love to see the overall section simplified, and I like your suggestion.

Reply:
Thanks, I will update